CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## [Unreleased]
 
+## [4.15.1]
+
+-  Make sure
+
+    1. Jabbax is loaded before we define `Surgex.Parser`
+    2. Plug is loaded before we define `Surgex.Sentry`
+
+to avoid compilation issues in apps that depend on Surgex.
```

Comment:
I know the function is `ensure_loaded?/1` but it's mostly used to verify if a module is available, so I'd reword this changelog:

```suggestion
-  Fix optional dependency on `Jabbax` and `Plug` by:

    1. defining `Surgex.Parser` only if `Jabbax` is available
    2. defining `Surgex.Sentry` only if `Plug` is available
```

Reply:
Wouldn't it try to load them if it's not there first anyway?
But the version you suggested makes more sense and looks better, so thank you!

Reply:
@andreyuhai yes, it tries to load them if not loaded yet, but in the end returns a boolean saying if it's available or not.

Reply:
Yeap, got it, thank you! 🙇